app/controllers/manage_proposal_controller.rb

```diff
@@ -98,20 +102,22 @@ def publish_delete_proposal(proposal, provider)
     if search_response.body['hits'].to_s != '1'
       # If the search has more than one hit or 0 hits, the record was not
       # uniquely identified from it's native ID.
-      flash[:error] = I18n.t('controllers.manage_proposals.publish.flash.delete.not_found_error')
+      flash[:error] = I18n.t('controllers.manage_proposal.publish_proposal.flash.delete.not_found_error')
       Rails.logger.info("Could not complete delete request from proposal with short name: #{proposal['short_name']} and id: #{proposal['id']} because it could not be located.")
     elsif !search_response.body['items'][0]['meta']['granule-count'].zero?
       # Do not allow the deletion of collections which have granules
-      flash[:error] = I18n.t('controllers.manage_proposals.publish.flash.delete.granules_error')
+      flash[:error] = I18n.t('controllers.manage_proposal.publish_proposal.flash.delete.granules_error')
       Rails.logger.info("Could not complete delete request from proposal with short name: #{proposal['short_name']} and id: #{proposal['id']} because it had granules at the time of the delete attempt.")
     else
       cmr_response = cmr_client.delete_collection(provider, proposal['native_id'], token)
 
       if cmr_response.success?
-        flash[:success] = I18n.t('controllers.manage_proposals.publish.flash.delete.success')
+        flash[:success] = I18n.t('controllers.manage_proposal.publish_proposal.flash.delete.success')
         Rails.logger.info("Audit Log: Collection with native_id #{proposal['native_id']} was deleted for #{provider} by #{session[:urs_uid]} by proposal with short name: #{proposal['short_name']} and id: #{proposal['id']}.")
+
+        update_proposal_status_in_dmmt(proposal)
       else
-        flash[:error] = I18n.t('controllers.manage_proposals.publish.flash.delete.error')
+        flash[:error] = I18n.t('controllers.manage_proposal.publish_proposal.flash.delete.error')
```

Comment:
I am thinking through if we should do something more like `flash[:error] = cmr_response.error_message(i18n: I18n.t('controllers.manage_proposal.publish_proposal.flash.delete.error'))`, and leaning towards yes.